src/applications/personalization/profile/util/contact-information/phoneUtils.js

```diff
@@ -11,7 +10,8 @@ export const phoneFormSchema = {
     },
     inputPhoneNumber: {
       type: 'string',
-      pattern: '^\\d{10}$',
+      pattern: '^[0-9- ]+$',
```

Comment:
Should we check for parentheses too?

Reply:
I added a check for parens @allisonlu thanks for pointing that out. I initially tried to pattern match to make sure the parens had 3 digits between but couldn't get that to work, so I just added parens as accepted chars in the regex.

good question, the code that removes the special characters is used when submitting the field to the api. It splits up the number and area code as well before sending to the api. @dellerbie

Reply:
If we allowed anything, then yes the stripping would get rid of any non-digits before sending to the api, but it was causing an issue with the form field being dirty / not dirty in regards to the "Edit" page usage of the phone field.

There are lost of weird issues like this that I am finding with using the form library.